README.md

```diff
@@ -21,27 +21,19 @@
 
 *Support for Table accounts in CosmosDB is coming soon!
 
-## Mongo "Scrapbooks"
+![Browse CosmosDB and MongoDB databases](resources/Browse.png)
 
-- Configure the user setting `mongo.shell.path` to your mongo shell executable path
-- Click on any DB to open the Mongo shell playground editor
-- Enter your scripts, eg: `db.<collectionName>.find()`
-- IntelliSense (completions) will be provided as you write your scripts
-- Select the script and press `CMD+"` (`CTRL+"` on Windows and Linux) to see the results
+## Mongo "Scrapbooks" (Run Mongo Commands with Rich Intellisense)
 
-## Managing Azure Subscriptions
-
-If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
-
-![Sign in to Azure](resources/SignIn.gif)
-
-If you don't have an Azure Account, you can sign up for one today for free and receive $200 in credits by selecting "Create a Free Azure Account..." or selecting "View->Command Palette" and searching for "Azure: Create an Account".
+- Attach to a MongoDB database account
+- Optionally configure the user setting `mongo.shell.path` to your mongo shell executable path (many of the common commands have built-in support and do not require the Mongo shell to be installed - see [Prerequisites](#prerequisites))
+- Right-click on a database and select "Connect"
+- Click on "New Mongo Scrapbook" in the tree title bar
+- Enter your script, eg: `db.<collectionName>.find()`
+- IntelliSense (completions) will be provided as you write your script
+- Select the script and press `CMD+"` (Mac) or `CTRL+"` (Windows and Linux) to see the results

```

Comment:
"Select" doesn't seem accurate to me. How about, "With your cursor inside the command, press ..."

Also we really need to address this bug: https://github.com/Microsoft/vscode-cosmosdb/issues/386 otherwise this bullet point only applies for Mac/Linux